packages/api-plugin-fulfillment-method-pickup-store/src/fulfillmentMethodsWithQuotesPickupStore.js

```diff
@@ -0,0 +1,75 @@
+import Logger from "@reactioncommerce/logger";
+import collectStoreDetails from "./util/collectStoreDetails.js";
+
+const packageName = "fulfillment-method-pickup-store";
+const fulfillmentTypeName = "pickup";
+const fulfillmentMethodName = "store";
+const logCtx = { name: "fulfillment-method-pickup-store", file: "fulfillmentMethodsWithQuotesPickupStore" };
+
+/**
+ * @summary Returns a list of fulfillment method quotes based on the items in a fulfillment group.
+ * @param {Object} context - Context
+ * @param {Object} commonOrder - details about the purchase a user wants to make.
+ * @param {Array} [previousQueryResults] - an array of shipping rates and
+ * info about failed calls to the APIs of some shipping methods providers
+ * @returns {Array} - an array that contains two arrays: the first array will
+ * be an updated list of shipping rates, and the second will contain info for
+ * retrying this specific package if any errors occurred while retrieving the
+ * shipping rates.
+ * @private
+ */
+export default async function fulfillmentMethodsWithQuotesPickupStore(context, commonOrder, previousQueryResults = []) {
```

Comment:
This will be updated after the initial merge to feat/fulfillment-type branch
Issue for tracking [here](https://app.zenhub.com/workspaces/moc-main-board-62fd8e881686130027eafc31/issues/gh/reactioncommerce/reaction/6821)